react/package.json

```diff
@@ -8,7 +8,7 @@
     "@testing-library/user-event": "^7.1.2",
     "express": "^4.17.1",
     "imagekit": "^3.1.0",
-    "imagekitio-react": "^1.0.8",
+    "imagekitio-react": "^2.0.0",
```

Comment:
Changing here won't be enough. You will have to actually bump so that all corresponding dependencies are updated in yarn.lock as well.